cpp/src/parquet/metadata.h

```diff
@@ -182,6 +184,28 @@ class PARQUET_EXPORT ColumnChunkMetaData {
   std::unique_ptr<ColumnChunkMetaDataImpl> impl_;
 };
 
+// \brief DataPageStats is a proxy around format::DataPageHeader and
+// format::DataPageHeaderV2.
+class PARQUET_EXPORT DataPageStats {
```

Comment:
I'm not sure why this needs to manually define accessors and constructor instead of being a plain struct?
```c++
struct DataPageStats {
  EncodedStatistics* encoded_statistics;
  int32_t num_values;
  std::optional<int32_t> num_rows;
};
```

Reply:
Done.